src/modal/alert-modal.component.ts

```diff
@@ -64,7 +64,7 @@ import Modal from "./modal.decorator";
       			<p class="bx--modal-header__heading bx--type-beta">{{modalTitle}}</p>
 			</ibm-modal-header>
 			<div class="bx--modal-content">
-				<p>{{modalContent}}</p>
+				<p [innerHTML]="modalContent"></p>
```

Comment:
I think this is probably okay (I can certainly see some use cases), but a `ng-template` output should probably be added as the default way to inject complex html content.

That of course could be added as a feature later, what do you think?

Reply:
Yep makes sense, and I think we already support that for banners/notifications (which have a similar usage model)